docs/contributing/docs-templates.md

```diff
@@ -187,29 +194,74 @@ Assume the reader has basic programming knowledge like the command line, code ed
 
 ---
 
-## Tutorial
+## Recipes
 
-### Near perfect example of a tutorial
+[Docs Recipes](/docs/recipes/) should act as discoverable, concise instructions for completing common Gatsby tasks without having to navigate elsewhere. A recipe should list requirements and a few short steps to complete a task, listing actionable instructions inline and omitting everything else. Recipes are smaller units than tutorials: multiple recipes could be linked from a reference guide or tutorial.
 
-[Main Gatsby tutorial](https://www.gatsbyjs.org/tutorial/)
+The components of a recipe are:
 
-### What should a tutorial be about?
+- The name of the recipe, which should describe a single task
+- Requirements and prerequisites
+- Step-by-step directions
+- A link to a working example
+
+Recipes should be short. If you're finding a recipe is becoming too long to fit on the Docs Recipes page due to including many prerequisites or steps, consider writing a tutorial instead.
 
-We need tutorials to explain how to perform a series of related tasks with Gatsby.
+### Recipe template
 
-### When to write a guide vs. a tutorial?
+````markdown
+## Recipe name
 
-A guide explains a discrete task without the step-by-step context provided by a tutorial.
+### Requirements
 
-Guides cover the smallest possible topic, while tutorials can cover a series of related tasks that you string together.
+- A Gatsby site with two page components: `index.js` and `contact.js`
+- The Gatsby <Link /> component
+- The Gatsby CLI method `gatsby develop`
+
+### Directions
+
+1. Open the index page component (src/pages/index.js), import the <Link />
+   component from Gatsby, add a <Link /> component above the header, and give
+   it a `to` property with the value of "/contact/" for the pathname:
+
+```jsx:title=src/pages/index.js
+import React from "react"
+import { Link } from "gatsby"
+
+export default () => (
+  <div style={{ color: `purple` }}>
+    <Link to="/contact/">Contact</Link>
+    <p>What a world.</p>
+  </div>
+)
+```
+
+2. Run `gatsby develop` and navigate to the index page. You should have a link
+   that takes you to the contact page when clicked!
+
+// optional:
+For a working example, check out **this example.**
+````
+
+## Tutorials
+
+In contrast to recipes and reference guides, tutorials are step-by-step instructions for a series of related Gatsby tasks. Tutorials are grouped into Gatsby Fundamentals and Intermediate Tutorials ("The Gatsby Tutorial"), as well as Advanced Tutorials which can be explored individually as needed.
```

Comment:
what does "grouped into" mean?

Reply:
Good question! It's in reference to the changes on the Tutorials page, where steps 0-8 are all part of the same flow:    https://github.com/gatsbyjs/gatsby/pull/14071